components/integration/production/base/kustomization.yaml

```diff
@@ -11,6 +11,11 @@ images:
   newName: quay.io/redhat-appstudio/integration-service
   newTag: b37833f64c8017c942ba95ca648d3b880b39af62
 
+configMapGenerator:
+- name: console-url
+  literals:
+    - CONSOLE_URL="https://console.redhat.com/preview/application-pipeline/ns/%s/pipelinerun/%s"
```

Comment:
I just tried this URL out, but it doesn't seem to work for me :/

Reply:
Here is an example for stage env.
https://console.dev.redhat.com/preview/application-pipeline/ns/mbasti-tenant/pipelinerun/test-python-gitlab-kmgvd-fhcws
It should redirects.

Reply:
Yup, you're right, I was using the prod instance and the redirect took ages, but it worked in the end.